_extras/snippets_library/UCL_sge/13/long_job.snip

```diff
@@ -1,6 +1,8 @@
-#!/bin/bash
-#PBS -l 0:0:30
+#!/bin/bash -l
+#$ -N long_job
+#$ -l h_rt=0:0:30
 
 echo 'This script is running on:'
 hostname
 sleep 120
+echo 'This script has finished successfully'
```

Comment:
I think that line should be also on all the other examples. It makes sense to see when something is done :)